mongomock/store.py

```diff
@@ -104,3 +107,51 @@ def __len__(self):
     def documents(self):
         for doc in six.itervalues(self._documents):
             yield doc
+
+    def remove_expired_documents(self):
+        for index in six.itervalues(self.indexes):
+            if index.get('expireAfterSeconds') is not None:
+                self._expire_documents(index)
+
+    def _expire_documents(self, index):
+        # Ignore non-integer values
```

Comment:
Please add a TODO to use a caching mechanism to avoid re-expiring the documents if we just did and that no document was added / updated.